src/components/Segmented/Segmented.utils.ts

```diff
@@ -0,0 +1,73 @@
+import { Option, OptionsAlignments } from './Segmented.types'
+
+const { Horizontal, Vertical } = OptionsAlignments
+
+/**
+ * Type guard function that checks if an element is of type string.
+ *
+ * @param {Option | string | number} element - The element to check.
+ * @returns {boolean} `true` if the option is a string, `false` otherwise.
+ */
+export function isString(element: Option | number): element is string {
+  return typeof element === 'string'
+}
+
+/**
+ * Checks if an option is disabled.
+ *
+ * @param {Option} option - The option to check.
+ * @param {boolean} isDisabled - Specifies whether the whole segmented is disabled.
+ *
+ * @returns {boolean} `true` if the option is disabled, `false` otherwise.
+ */
+export function isDisabledOption(option: Option, isDisabled = false): boolean {
+  return isDisabled ?? (
```

Comment:
The nullish operator returns the right-hand value if the left-hand value is null or undefined. 
In this case, the default value is false. 
So this function will return false if the value is undefined. 
How it works? Am I missing something?

Reply:
I changed it, now the function accept a general boolean isDisabled without default prop